### PR TITLE
docs: document variable descriptions

### PR DESCRIPTION
**Pull request by zhpenkov**
Documents the variable descriptions of the following components:

- DropZone
- Captcha
- Rating
- FileManager
- PDFViewer

### CODE REVIEW

File: packages/bootstrap/docs/customization-pdf-viewer.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+    </td>
+</tr>
+<tr>
+    <td>$kendo-pdf-viewer-toolbar-text</td>
+    <td>Null</td>
+    <td><code>null</code></td>
+    <td><code>null</code></td>
+</tr>
+<tr>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The text color of the PDFViewer toolbar.</div></div>
```
Comment (yordan-mitev) [inline packages/bootstrap/docs/customization-pdf-viewer.md:117]: PDFViewer Toolbar

File: packages/bootstrap/docs/customization.md [hunk 2]
```diff
@@ -5380,7 +5540,7 @@ The following table lists the available variables for customizing the Bootstrap
 </tbody>
 </table>
 
-### Editor
+### Dropzone
```
Comment (yordan-mitev) [inline packages/bootstrap/docs/customization.md:5543]: Either Dropzone or DropZone in all docs

File: packages/bootstrap/docs/customization.md [hunk 3]
```diff
@@ -5830,7 +6200,7 @@ The following table lists the available variables for customizing the Bootstrap
 </tbody>
 </table>
 
-### Filter
+### Filemanager
```
Comment (yordan-mitev) [inline packages/bootstrap/docs/customization.md:6203]:
```suggestion
### FileManager
```

File: packages/bootstrap/scss/pdf-viewer/_variables.scss [hunk 4]
```diff
@@ -1,44 +1,111 @@
 // PDF viewer
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/pdf-viewer/_variables.scss:1]:
```suggestion
// PDFViewer
```

File: packages/classic/docs/customization-dropzone.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
+---
+title: Customizing Dropzone
```
Comment (yordan-mitev) [inline packages/classic/docs/customization-dropzone.md:2]: Choose one case style.

File: packages/classic/docs/customization-filemanager.md [hunk 6]
```diff
@@ -0,0 +1,608 @@
+---
+title: Customizing Filemanager
```
Comment (yordan-mitev) [inline packages/classic/docs/customization-filemanager.md:2]:
```suggestion
title: Customizing FileManager
```

File: packages/classic/docs/customization-filemanager.md [hunk 7]
```diff
@@ -0,0 +1,608 @@
...
+description: "Refer to the list of the Kendo UI Classic theme variables available for customization."
+slug: variables_kendothemeclassic_filemanager
+position: 9
+---
+
+# Customizing Filemanager
```
Comment (yordan-mitev) [inline packages/classic/docs/customization-filemanager.md:8]:
```suggestion
# Customizing FileManager
```

File: packages/classic/docs/customization-filemanager.md [hunk 8]
```diff
@@ -0,0 +1,608 @@
...
+    </td>
+</tr>
+<tr>
+    <td>$kendo-file-manager-toolbar-border-width</td>
+    <td>Number</td>
+    <td><code>$kendo-file-manager-border-width</code></td>
+    <td><code>1px</code></td>
+</tr>
+<tr>
+    <td colspan="4" class="theme-variables-description-container"><div><b>Description</b><div class="theme-variables-description">The border width of the FileManager toolbar.</div></div>
```
Comment (yordan-mitev) [inline packages/classic/docs/customization-filemanager.md:117]: Consider the case for the components integrated in the FileManager. For example, Toolbar, ListView, etc.

File: packages/classic/docs/customization-pdf-viewer.md [hunk 9]
```diff
@@ -0,0 +1,368 @@
+---
+title: Customizing Pdf-viewer
```
Comment (yordan-mitev) [inline packages/classic/docs/customization-pdf-viewer.md:2]:
```suggestion
title: Customizing PDFViewer
```

File: packages/classic/scss/filemanager/_variables.scss [hunk 10]
```diff
@@ -1,67 +1,182 @@
 // File manager
```
Comment (yordan-mitev) [inline packages/classic/scss/filemanager/_variables.scss:1]:
```suggestion
// FileManager
```

File: packages/bootstrap/docs/customization-pdf-viewer.md [hunk 11]
```diff
@@ -0,0 +1,368 @@
+---
+title: Customizing Pdf-viewer
```
[the remark this answers is not in the document]
Comment (zhpenkov) [reply packages/bootstrap/docs/customization-pdf-viewer.md:2]: The title field is autogenerated.

File: packages/bootstrap/docs/customization-pdf-viewer.md [hunk 12]
```diff
@@ -0,0 +1,368 @@
...
+description: "Refer to the list of the Kendo UI Bootstrap theme variables available for customization."
+slug: variables_kendothemebootstrap_pdf-viewer
+position: 9
+---
+
+# Customizing Pdf-viewer
```
[the remark this answers is not in the document]
Comment (zhpenkov) [reply packages/bootstrap/docs/customization-pdf-viewer.md:8]: The customizing field is autogenerated.

**Review by yordan-mitev — APPROVED** [review]

File: packages/bootstrap/scss/pdf-viewer/_variables.scss [hunk 13]
```diff
@@ -1,44 +1,111 @@
...
+/// @group pdf-viewer
 $kendo-pdf-viewer-font-size: $kendo-font-size-md !default;
+/// The line height of the PDFViewer.
+/// @group pdf-viewer
 $kendo-pdf-viewer-line-height: $kendo-line-height-md !default;
 
+/// The background color of the PDFViewer.
+/// @group pdf-viewer
 $kendo-pdf-viewer-bg: $kendo-component-bg !default;
+$kendo-pdf-viewer-bg: var( --kendo-component-bg, initial ) !default;
```
Comment (epetrow) [inline packages/bootstrap/scss/pdf-viewer/_variables.scss:19]: This line should be removed, seems like a copy paste mistake

File: packages/classic/scss/pdf-viewer/_variables.scss [hunk 14]
```diff
@@ -1,44 +1,111 @@
...
+/// @group pdf-viewer
 $kendo-pdf-viewer-font-size: $kendo-font-size-md !default;
+/// The line height of the PDFViewer.
+/// @group pdf-viewer
 $kendo-pdf-viewer-line-height: $kendo-line-height-md !default;
 
+/// The background color of the PDFViewer.
+/// @group pdf-viewer
 $kendo-pdf-viewer-bg: $kendo-component-bg !default;
+$kendo-pdf-viewer-bg: var( --kendo-component-bg, initial ) !default;
```
Comment (epetrow) [inline packages/classic/scss/pdf-viewer/_variables.scss:19]: Remove this line here also

File: packages/default/scss/pdf-viewer/_variables.scss [hunk 15]
```diff
@@ -1,44 +1,111 @@
...
+/// @group pdf-viewer
 $kendo-pdf-viewer-font-size: $kendo-font-size-md !default;
+/// The line height of the PDFViewer.
+/// @group pdf-viewer
 $kendo-pdf-viewer-line-height: $kendo-line-height-md !default;
 
+/// The background color of the PDFViewer.
+/// @group pdf-viewer
 $kendo-pdf-viewer-bg: $kendo-component-bg !default;
+$kendo-pdf-viewer-bg: var( --kendo-component-bg, initial ) !default;
```
Comment (epetrow) [inline packages/default/scss/pdf-viewer/_variables.scss:19]: Remove here as well

File: packages/material/scss/pdf-viewer/_variables.scss [hunk 16]
```diff
@@ -1,44 +1,111 @@
...
+/// @group pdf-viewer
 $kendo-pdf-viewer-font-size: $kendo-font-size-md !default;
+/// The line height of the PDFViewer.
+/// @group pdf-viewer
 $kendo-pdf-viewer-line-height: $kendo-line-height-md !default;
 
+/// The background color of the PDFViewer.
+/// @group pdf-viewer
 $kendo-pdf-viewer-bg: $kendo-component-bg !default;
+$kendo-pdf-viewer-bg: var( --kendo-component-bg, initial ) !default;
```
Comment (epetrow) [inline packages/material/scss/pdf-viewer/_variables.scss:19]: Remove this line here as well

**Review by epetrow — APPROVED** [review]

**Pull request merged**